### ISSUE TITLE
Update and Organize the Main Figma Document

### ISSUE THREAD

**Issue by KPHowley** [hackforla/tdm-calculator#537]
### Overview
Our Main figma document is a little disjointed and hard to follow. We want to update this and make it easier to follow and contribute to. 

### Action Items

- [ ] Investigate documents from other teams , work with the UI/UX COP
- [ ] determine best origination system
- [ ] rename and organize pages
- [ ] create a single page or easy way to identify current versions 

### Resources/Instructions
Sample documents from other projects:
https://www.figma.com/file/0RRPy1Ph7HafI3qOITg0Mr/Hack-for-LA-Website

https://www.figma.com/file/BJL6ZHfjrxVhU8jvD9gB3P/HomeUniteUs-Everything-Figma?node-id=1%3A934

**Comment by mojimoh** [comment]
- Merging with #1144, update action items on that issue.
